### ISSUE TITLE
Grid unselect event fires the select event

### ISSUE THREAD

**Issue by NoelSMartin** [vitmalina/w2ui#2249]
**Short description**
select event is fired on grid unselect

**What is current behavior**
Using current master https://rawgit.com/vitmalina/w2ui/master/dist/w2ui.min.js
select event is fired on grid unselect

**What is desired behavior**
unselect event is fired

**Link to jsfidle/codepan with sample code**
https://jsfiddle.net/Ltfjhk0w/

**Steps to reproduce or sample** 
...

**Comment by vitmalina** [comment]
Not a bug, but an intended change. Over the years, I realized that unselect events are not very useful. I do not normally care if a record is unselected, what I care about is if selection changed. Plus firing separate event for each unselected record could lead to too many events. So, I refactored the code and updated documentation and examples, see https://w2ui.com/web/docs/2.0/w2grid.onSelect

In short, `select` event is fired when selection changes.
